### PR TITLE
Allow users to specify conntrack buffer size.

### PR DESCRIPTION
**Pull request by tomwilkie**
So they can work around the ENOBUFFs problems.

### CODE REVIEW

File: probe/endpoint/conntrack_internal_test.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 )
 
 const conntrackCloseTag = "</conntrack>\n"
+const bufferSize = 1024 * 1024
```
Comment (2opremio) [inline probe/endpoint/conntrack_internal_test.go]: Have you tested this value in dev/prod? (to confirm the default doesn't value is large enough for at least our usecase)

File: prog/probe.go [hunk 2]
```diff
@@ -152,7 +152,7 @@ func probeMain(flags probeFlags) {
 		defer dnsSnooper.Stop()
 	}
 
-	endpointReporter := endpoint.NewReporter(hostID, hostName, flags.spyProcs, flags.useConntrack, flags.procEnabled, flags.procRoot, scanner, dnsSnooper)
+	endpointReporter := endpoint.NewReporter(hostID, hostName, flags.spyProcs, flags.useConntrack, flags.procEnabled, flags.procRoot, flags.conntrackBufferSize, scanner, dnsSnooper)
```
Comment (2opremio) [inline prog/probe.go]: Time for a config datastructure? (the number of parameters is getting out of hand)

**Comment by 2opremio** [comment]
Minor comment/suggestion. Otherwise LGTM

File: probe/endpoint/conntrack_internal_test.go (hunk 1 above)
Comment (tomwilkie) [inline probe/endpoint/conntrack_internal_test.go]: This is just for a test, and is actually ignored.

File: prog/probe.go (hunk 2 above)
Comment (tomwilkie) [inline prog/probe.go]: Yeah I was thinking that too...

Comment (tomwilkie) [inline prog/probe.go]: Done.

**Comment by 2opremio** [comment]
LGTM

File: prog/main.go [hunk 3]
```diff
@@ -197,6 +199,7 @@ func main() {
 
 	// Proc & endpoint
 	flag.BoolVar(&flags.probe.useConntrack, "probe.conntrack", true, "also use conntrack to track connections")
+	flag.IntVar(&flags.probe.conntrackBufferSize, "probe.conntrack.buffersize", 208*1024, "conntrack buffer size")
```
Comment (rade) [inline prog/main.go]: where did the default value come from?

Comment (rade) [inline prog/main.go]: looks like that's the default, on some Linuxes at least...
```
$ cat /proc/sys/net/core/rmem_default
212992
```

**Comment by tomwilkie** [comment]
Yeah I read it off the Dev cluster and my mini kube VM.

**Pull request merged**